target/linux/armsr/modules.mk

```
@@ -287,6 +287,23 @@ define KernelPackage/octeontx2-net
 endef
 $(eval $(call KernelPackage,octeontx2-net))
 
+define KernelPackage/renesas-net-avb
+SUBMENU:=$(NETWORK_DEVICES_MENU)
+TITLE:=Renesas network drivers
+DEPENDS:=@(TARGET_armsr_armv8) +kmod-phylink +kmod-mii +kmod-ptp +kmod-libphy +kmod-mdio-gpio
+KCONFIG:=CONFIG_NET_VENDOR_RENESAS=y \
+    CONFIG_RAVB \
+    CONFIG_PTP_1588_CLOCK_OPTIONAL=y 
+FILES=$(LINUX_DIR)/drivers/net/ethernet/renesas/ravb.ko
+AUTOLOAD=$(call AutoLoad,40,ravb.ko)
```

```suggestion
AUTOLOAD:=$(call AutoProbe,ravb)
```
Please use AutoProbe and do not add a .ko here.